docs/README.md

```diff
@@ -865,7 +865,87 @@ _Reference_
 - https://radiant-brushlands-42789.herokuapp.com/medium.com/better-programming/javascript-es2020-features-with-simple-examples-d301dbef2c37
 - https://www.freecodecamp.org/news/javascript-new-features-es2020/
 
-<br/> <br/> <br/>
+<br/>
+
+## 17. DOM Manipulation
+
+Document Object Model, or DOM for short, represents all page content as objects that can be modified. And Javascript allow us to modify DOM.
+
+With JavaScript you can interact with the DOM to:
+- inspect the page structure
+- access the page metadata and headers
+- edit the CSS styling
+- attach or remove event listeners
+- edit any node in the page
+- change any node attribute
+- .. and much more.
+
+The main 2 objects provided by the DOM API that you you will interact the most with are document and window.
+
+ 1.  The Window object - The window object represents the window that contains the DOM document.
+window.document points to the document object loaded in the window.
+Properties and methods of this object can be called without referencing window explicitly, because it represents the global object. So, the previous property window.document is usually called just document.
+
```

Comment:
Here it is mentioned that "The main 2 objects provided by the DOM API that **you you** will interact the most with are document and window."
Repeated **you**

Also add a small note on the document as the 2nd point
(It would be great if you could mention the diff btw document and window aslo)

Reply:
Done.